sample-data/image-segments/Array0To9.st

```diff
@@ -0,0 +1 @@
+^ { 1 . 2. 3 . 4 . 5 . 6 . 7 . 8 . 9 }.
```

Comment:
There are some spacinnissues here.

Also either change the file name or add 0 to the array.